httpclient5/src/main/java/org/apache/hc/client5/http/config/RequestConfig.java

```diff
@@ -533,23 +533,25 @@ public Builder setConnectTimeout(final long connectTimeout, final TimeUnit timeU
         }
 
         /**
-         * Determines the timeout until arrival of a response from the opposite
-         * endpoint.
-         * <p>
+         * Determines the timeout until arrival of a response from the opposite endpoint.
          * A timeout value of zero is interpreted as an infinite timeout.
-         * </p>
          * <p>
-         * Please note that response timeout may be unsupported by
-         * HTTP transports with message multiplexing.
+         * Please note that response timeout may be unsupported by HTTP transports with
+         * message multiplexing.
          * </p>
          * <p>
-         * Please note that response timeout is not a deadline. Its absolute value
-         * can be exceeded, for example, in case of automatic request re-execution.
-         * Please make sure the automatic request re-execution policy has been
-         * configured appropriately.
+         * Please note that response timeout is not a deadline. Its absolute value can
+         * be exceeded, for example, in case of automatic request re-execution. Please
+         * make sure the automatic request re-execution policy has been configured
+         * appropriately.
          * </p>
          * <p>
-         * Default: {@code null}
+         * In classic (blocking) I/O execution, if this value is set, it will be applied
```

Comment:
@LichKing-lee Same here. You can mention that this parameter can override the setting of the connection layer or the i/o layer but `ConnectionConfig` is of no relevance here